triplestore/demo/demo.py

```diff
@@ -27,13 +27,15 @@ def main() -> None:
         description="Unified triplestore demo showcasing the abstraction layer."
     )
     parser.add_argument(
+        "--backend",
         "-b",
         choices=["allegrograph", "graphdb", "blazegraph", "jena", "oxigraph"],

```

Comment:
This defies the "dynamic registration" bit, doesn't it?
And makes the next `if` (line 40) impossible to succeed.